cbtdoc.go

```diff
@@ -104,6 +104,22 @@ options to your ~/.cbtrc file in the following format:
 
 All values are optional and can be overridden at the command prompt.
 
+Add a value to an aggregate cell (write)
```

Comment:
I think you're missing the `#` here: `# Add a value to an aggregate cell (write)`